demo/src/App.js

```diff
@@ -11,6 +11,7 @@ const MIN_ZOOM = -8;
 const DEFAULT_VIEW_STATE = { zoom: -5.5, target: [30000, 10000, 0] };
 const COLORMAP = 'viridis';
 const COLORMAP_SLIDER_CHECKBOX_COLOR = [220, 220, 220];
+const OFFSET = 25;
```

Comment:
Sorry for being late on commenting on this. In CSS, `margin` or `padding` usually describes this effect. Do you think that would be an easier term for users to understand?

Reply:
I'm being pedantic so hopefully we never need to revisit :)

Reply:
Sure, we can do `margin`